product_attribute_model_link/models/product_attribute_value.py

```diff
@@ -174,9 +191,7 @@ def action_open_linked_record_wizard(self):
         self.ensure_one()
         context = {
             "default_linked_record_ref": (
```

Comment:
Why to keep that field in wizard then?

Reply:
Since everything was refactored, I don't really see the rationale to keep it like that in the wizard - yet I may be missing something

Reply:
I think that the single reference field in the wizard significantly simplifies the user experience by streamlining the selection process. It makes it easier for users to navigate and choose the appropriate record without the complexity of multiple fields or the need to input IDs directly. Do you have any concerns or alternative perspectives on this approach? I'm open to hearing your thoughts and discussing any other ideas you might have.